Dockerfile

```diff
@@ -31,11 +31,27 @@ RUN apt-get update -qq \
     && rm *.deb
 
 ## Install Packages
-RUN Rscript -e "install.packages(c('littler', 'docopt', 'tidyverse', 'sparklyr'), repo = 'http://cran.us.r-project.org')"
+RUN mlserver-Rscript-bin -e "install.packages(c('littler', 'docopt', 'tidyverse', 'sparklyr'), repo = 'http://cran.us.r-project.org')"
 
 ## Install rstudio-server
 RUN wget https://download2.rstudio.org/rstudio-server-1.1.383-amd64.deb
 RUN gdebi rstudio-server-1.1.383-amd64.deb --non-interactive
 RUN echo "server-app-armor-enabled=0" | tee -a /etc/rstudio/rserver.conf
 
+# Create user
+RUN set -e \
+      && useradd -m -d /home/rstudio rstudio \
+      && echo rstudio:rstudio \
+        | chpasswd
+
+# Set R to R Client
+
+# set rclient to rsession for rstudio-server
+RUN echo "rsession-which-r=/opt/microsoft/rclient/3.4.1/bin/R/R" >> /etc/rstudio/rserver.conf
+RUN echo "r-libs-user=/opt/microsoft/rclient/3.4.1/libraries/RServer" >> /etc/rstudio/rsession.conf
+
+# start rstudio-server session
 EXPOSE 8787
+
+CMD ["/usr/lib/rstudio-server/bin/rserver", "--server-daemonize=0", "--server-app-armor-enabled=0"]
```

Comment:
For now, we can remove the start rserver script and use a custom script to start the rstudio server only on the master node